service/history/historyEngine.go

```diff
@@ -318,9 +318,9 @@ func (e *historyEngineImpl) StartWorkflowExecution(startRequest *h.StartWorkflow
 		}
 		replicationTasks = append(replicationTasks, replicationTask)
 	}
+	setTaskVersion(msBuilder.GetCurrentVersion(), transferTasks, timerTasks)
```

Comment:
here it is outside the create workflow handler and down below it is inside the handler.  Why the difference?

Reply:
good catch, copy paste error